frontend/lib/src/components/core/Block/ElementNodeRenderer.tsx

```diff
@@ -246,6 +246,7 @@ const RawElementNodeRenderer = (
     ...elementProps,
     widgetMgr: props.widgetMgr,
     disabled: props.widgetsDisabled,
+    fragmentId: node.fragmentId,
```

Comment:
This line (and a few changes at the very bottom of this file) is essentially the only meaningful change in this file.

The rest of it is a drive-by sorting/grouping of widgets/elements into alphabetical order